post/get_received_message.js

```diff
@@ -197,9 +232,18 @@ module.exports = ({description, lnd, payments, received}, cbk) => {
       // Final received message
       message: [
         'messageDetails',
+        'parseBalancedOpen',
         'receivedMessage',
-        ({messageDetails, receivedMessage}, cbk) =>
+        ({messageDetails, parseBalancedOpen, receivedMessage}, cbk) =>
       {
+        if (!!parseBalancedOpen) {
+          const message = `Received a ${escape(parseBalancedOpen.capacity)} Balanced Channel Open request from ${escape(parseBalancedOpen.alias)} \`${parseBalancedOpen.destination}\``;
```

Comment:
Potentially the alias is empty, could be `alias || id.substring(0, 8)` or something like that?

Reply:
fixed it.